workbench-core/accounts/src/services/accountService.ts

```diff
@@ -9,20 +9,28 @@ import Boom from '@hapi/boom';
 import _ from 'lodash';
 import { HostingAccountStatus } from '../constants/hostingAccountStatus';
 
-interface Account {
-  id: string | undefined;
+export interface CostCenter {
```

Comment:
Wonder if we should consider move this and interface below under ``models` folder

Reply:
For now I left that to be decided when we start the `CostCenter` epic, but ya i agree, i just don't know where that folder will end up being and i figured it'd be easier to just do it when we start building out cost center infrastructure.